style.css/general.css

```diff
@@ -0,0 +1,33 @@
+html,
+body {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  font-family: "Montserrat", sans-serif;
+}
+
+.content {
+  flex: 1 0 auto;
+}
+
+.site-wrapper {
+  max-width: 1200px;
+  margin: 0 auto;
```

Comment:
Please add padding: 0 16px; to your site wrapper. The reason behind it we want our content to have some space from the device edge. I'll send some screen shot to make it more clear.